source/_templates/installations/elastic/common/copy_certificates_filebeat_wazuh_cluster.rst

```diff
@@ -1,6 +1,8 @@
 .. Copyright (C) 2020 Wazuh, Inc.
 
-For using the following command, the ``X`` must be replaced with the number defined for this Wazuh server in the ``instances.yml`` file:
+In section **Installing Elasticsearch**, the ``cert.zip`` file was created. The file must be copied into the Wazuh server host, for example, with a use of ``scp``. This guide assumes that the file is placed in ~/ (home user folder).
```

Comment:
It should be "with an use of", nevertheless, I think that in this context "use" is uncountable, so maybe you could use "for example, using scp".